rktl_autonomy/src/rktl_autonomy/_ros_interface.py

```diff
@@ -85,8 +101,12 @@ def __init__(self, node_name='gym_interface', eval=False, launch_file=None, laun
 
         # additional set up for training
         if not self.__EVAL_MODE:
-            self.__DELTA_T = rospy.Duration.from_sec(1.0 / rospy.get_param('~rate', 30.0))
-            self.__clock_pub = rospy.Publisher('/clock', Clock, queue_size=1, latch=True)
+            self.__DELTA_T = rospy.Duration.from_sec(
+                1.0 / rospy.get_param("~rate", 30.0)
+            )
+            self.__clock_pub = rospy.Publisher(
+                "/clock", Clock, queue_size=1, latch=True
+            )
```

Comment:
I'm not sure if I like the auto formatting. Personally, I think this is a step down in this particular instance.